docker/maistra-builder_2.1-1.9.Dockerfile

```diff
@@ -0,0 +1,148 @@
+FROM fedora:33
+
+# Versions
+ENV ISTIO_TOOLS_SHA=f4cf625fd815a0300e5ca541d03bc57057dad289
+ENV KUBECTL_VERSION="v1.20.0"
+ENV HELM3_VERSION=v3.1.2
+ENV VALE_VERSION="v2.1.1"
+ENV KIND_VERSION="v0.9.0"
```

Comment:
I think we need to update the `KIND_VERSION` here because v0.10.0 added support for k8 1.20. This will match with `KUBECTL_VERSION`.

Reply:
good point- but I think 0.9.0 might be safer as [that's what upstream uses for 1.9](https://github.com/istio/tools/blob/2a6f87c08f560190a554c257bf824bef15156a4e/docker/build-tools/Dockerfile#L54)